gamemodes/terrortown/gamemode/client/cl_equip.lua

```diff
@@ -390,16 +390,16 @@ local function CreateEquipmentList(t)
 	end
 
 	if #itms == 0 and not t.notalive then
-		ply:ChatPrint("[TTT2][SHOP] You need to run 'shopeditor' as admin in the developer console to create a shop for this role. Link it with another shop or click on the icons to add weapons and items to the shop.")
+		client:ChatPrint("[TTT2][SHOP] You need to run 'shopeditor' as admin in the developer console to create a shop for this role. Link it with another shop or click on the icons to add weapons and items to the shop.")
```

Comment:
just FYI, the command doesn't exist anymore.

Reply:
could you please rewrite this sentence to reflect the new UI? Then it is fine from my POV

Reply:
This is not reworked yet, so I would do that in another PR.
I just changed all plys to clients in one run.